front/public/locales/en/operationalStudies/manageTrainSchedule.json

```diff
@@ -31,7 +31,7 @@
     "noTrainStep": "The increment step of the train must be greater than zero",
     "noTrainStepInput": "Must be > 0",
     "softErrorInfra": "The application is unavailable, a server error has occurred, please try again later.",
-    "stdcmError": "Calculation error for last minute train path",
+    "stdcmError": "Calculation error for last minute train path: incompatibility with other train paths",
```

Comment:
You are not changing the message for the french language ?

Reply:
Mod did not pass. Corrected